where-to-travel/src/main/webapp/signup.js

```diff
@@ -0,0 +1,59 @@
+// Get elements for authentication
+const emailSignUp = document.getElementById('emailSignUp');
```

Comment:
Prefer to use CONSTANT_CASE: https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
Should they all be constant case?

Reply:
i believe so

Reply:
Since you could log in with a different email I don't think that this should be constant case. Jasmine sent me this link and I found it very helpful for understanding constant case. https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/